src/index.js

```diff
@@ -27,9 +27,9 @@ axios.interceptors.request.use(response => response,
 const logger = store => {
   return next => {
     return action => {
-      console.log('middleware', action);
+      console.log('[Middleware] dispatching', action);
       const result = next(action);
-      console.log(store.getState())
+      console.log('[Middleware] next state', store.getState())
       return result;
     }
   }
```

Comment:
When create a middleware - ideally to declare it in a separate file, out of enter point in order to keep it clean. Also a good practice would be to submit a separate folder "utils" or "middleware" for that purpose and keep it there.